docs/swagger.json

```diff
@@ -522,6 +522,39 @@
                 ]
             }
         },
+        "/mentorship_relations/count_pending": {
```

Comment:
I don't think we should modify swagger manually but will confirm that with other mentors